README.md

```diff
@@ -449,6 +444,15 @@ _your own configuration lines_
 """
 ```
 
+`plone.meta` generates a list of Python and Plone version combinations to run the distribution tests.
+
+You can customize that by defining your testing matrix:
+
+```toml
+[tox]
+test_matrix = { "6.1" = ["3.13", "3.10"], "6.0": ["3.13", "3.9"] }
+```
```

Comment:
We should say a bit more explicitly how this works. It's not obvious which one is the Plone version and which one is the Python version unless you recognize the numbers.

Reply:
Oh, I think I mention it on the upgrade docs but not on the readme indeed.